BeamAdapterConfig.cmake.in

```diff
@@ -4,6 +4,7 @@
 
 set(BEAMADAPTER_HAVE_SOFAIMPLICITFIELD @BEAMADAPTER_HAVE_SOFAIMPLICITFIELD@)
 set(BEAMADAPTER_HAVE_SOFADVANCEDCONSTRAINT @BEAMADAPTER_HAVE_SOFADVANCEDCONSTRAINT@)
+set(BEAMADAPTER_HAVE_SOFACUDA @BEAMADAPTER_HAVE_SOFACUDA@)
```

Comment:
How is this variable defined?

Reply:
it is done discretely by the sofa_find_package() macro